<ui version="4.0" >
 <class>KWFrameGeometry</class>
 <widget class="QWidget" name="KWFrameGeometry" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="leftMargin" >
    <number>9</number>
   </property>
   <property name="topMargin" >
    <number>9</number>
   </property>
   <property name="rightMargin" >
    <number>9</number>
   </property>
   <property name="bottomMargin" >
    <number>9</number>
   </property>
   <item>
    <widget class="QCheckBox" name="protectSize" >
     <property name="text" >
      <string>Protect size and position</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="positionGB" >
     <property name="title" >
      <string>Position</string>
     </property>
     <layout class="QGridLayout" >
      <property name="leftMargin" >
       <number>9</number>
      </property>
      <property name="topMargin" >
       <number>9</number>
      </property>
      <property name="rightMargin" >
       <number>9</number>
      </property>
      <property name="bottomMargin" >
       <number>9</number>
      </property>
      <property name="horizontalSpacing" >
       <number>6</number>
      </property>
      <property name="verticalSpacing" >
       <number>6</number>
      </property>
      <item rowspan="2" row="0" column="5" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Left:</string>
        </property>
        <property name="buddy" >
         <cstring>m_frameLeft</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Width:</string>
        </property>
        <property name="buddy" >
         <cstring>m_frameWidth</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="3" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>Top:</string>
        </property>
        <property name="buddy" >
         <cstring>m_frameTop</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="3" >
       <widget class="QLabel" name="label_5" >
        <property name="text" >
         <string>Height:</string>
        </property>
        <property name="buddy" >
         <cstring>m_frameHeight</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="KoUnitDoubleSpinBox" name="left" />
      </item>
      <item row="0" column="4" >
       <widget class="KoUnitDoubleSpinBox" name="top" />
      </item>
      <item row="1" column="2" >
       <widget class="KoUnitDoubleSpinBox" name="width" />
      </item>
      <item row="1" column="4" >
       <widget class="KoUnitDoubleSpinBox" name="height" />
      </item>
      <item rowspan="2" row="0" column="0" >
       <widget class="KWPositionSelector" native="1" name="positionSelector" />
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="marginsGB" >
     <property name="title" >
      <string>Margins</string>
     </property>
     <layout class="QGridLayout" >
      <property name="leftMargin" >
       <number>9</number>
      </property>
      <property name="topMargin" >
       <number>9</number>
      </property>
      <property name="rightMargin" >
       <number>9</number>
      </property>
      <property name="bottomMargin" >
       <number>9</number>
      </property>
      <property name="horizontalSpacing" >
       <number>6</number>
      </property>
      <property name="verticalSpacing" >
       <number>6</number>
      </property>
      <item rowspan="2" row="1" column="4" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>144</width>
          <height>41</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0" colspan="4" >
       <widget class="QCheckBox" name="synchronize" >
        <property name="text" >
         <string>Synchronize changes</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" >
       <widget class="QLabel" name="label_9" >
        <property name="text" >
         <string>Bottom:</string>
        </property>
        <property name="buddy" >
         <cstring>m_bottomMargin</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QLabel" name="label_8" >
        <property name="text" >
         <string>Top:</string>
        </property>
        <property name="buddy" >
         <cstring>m_topMargin</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_7" >
        <property name="text" >
         <string>Right:</string>
        </property>
        <property name="buddy" >
         <cstring>m_rightMargin</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_6" >
        <property name="text" >
         <string>Left:</string>
        </property>
        <property name="buddy" >
         <cstring>m_leftMargin</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="KoUnitDoubleSpinBox" name="leftMargin" />
      </item>
      <item row="2" column="1" >
       <widget class="KoUnitDoubleSpinBox" name="rightMargin" />
      </item>
      <item row="1" column="3" >
       <widget class="KoUnitDoubleSpinBox" name="topMargin" />
      </item>
      <item row="2" column="3" >
       <widget class="KoUnitDoubleSpinBox" name="bottomMargin" />
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>360</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KoUnitDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>KoUnitDoubleSpinBox.h</header>
  </customwidget>
  <customwidget>
   <class>KWPositionSelector</class>
   <extends>QWidget</extends>
   <header>dialogs/KWPositionSelector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
